roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -38,32 +38,81 @@
   when: create_ha_unit_files | changed
 # end workaround for missing systemd unit files
 
+- name: Preserve Master API Proxy Config options
+  command: grep PROXY /etc/sysconfig/{{ openshift.common.service_type }}-master-api
+  register: master_api_proxy
+  when: openshift.master.ha is defined and openshift.master.ha | bool and openshift_master_cluster_method == "native"
+      and upgrading is defined and upgrading | bool
```

Comment:
Unfortunately it is, @detiber explained the reasons why we don't want to
require that state in the roles. It went into master temporarily due to
time constraints. Can we find another way to tell the role what we want
here?
On Jun 3, 2016 4:45 PM, "Scott Dodson" <notifications@github.com> wrote:

> In roles/openshift_master/tasks/systemd_units.yml
> <https://github.com/openshift/openshift-ansible/pull/1964#discussion_r65764978>
> :
>
> > @@ -38,32 +38,81 @@
> >    when: create_ha_unit_files | changed
> >  # end workaround for missing systemd unit files
> >
> > +- name: Preserve Master API Proxy Config options
> > +  command: grep PROXY /etc/sysconfig/{{ openshift.common.service_type }}-master-api
> > +  register: master_api_proxy
> > +  when: openshift.master.ha is defined and openshift.master.ha | bool and openshift_master_cluster_method == "native"
> > +      and upgrading is defined and upgrading | bool
>
> @dgoodwin <https://github.com/dgoodwin> are you ok with my use of
> upgrading variable here? That's not going away in your in-progress work is
> it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1964/files/ac05c9d9127162d61a7b579a4178140da9171b1e#r65764978>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADIQcuQGiIKVjHGJhtA0uJLqE_JLyE_ks5qIIREgaJpZM4IqzUD>
> .
>